src/woo_publications/publications/api/filters.py

```diff
@@ -2,7 +2,38 @@
 
 from django_filters.rest_framework import FilterSet, filters
 
-from ..models import Publication
+from ..models import Document, Publication
+
+
+class DocumentFilterSet(FilterSet):
+    # TODO: change this filter to custom named filter with `@extend_schema_field(UUID)` once bug is fixed in drf-spectacular
+    publicatie = filters.ModelChoiceFilter(
+        queryset=Publication.objects.all(),
+        to_field_name="uuid",
+        help_text=_(
+            "Search the document based on the unique identifier (UUID) that represents a publication. "
+            "<sup><sub>Disclaimer: disregard the documented type `integer` the correct type is `UUID`.</sub></sup>"
```

Comment:
what's up with the sup/sub? :P

You can use markdown in these help texts and it will be rendered properly :)